src/Forms/Prism.Forms/Navigation/NavigationException.cs

```diff
@@ -12,6 +12,7 @@ public class NavigationException : Exception
         public const string IConfirmNavigationReturnedFalse = "IConfirmNavigation returned false";
         public const string NoPageIsRegistered = "No Page has been registered with the provided key";
         public const string ErrorCreatingPage = "An error occurred while resolving the page. This is most likely the result of invalid XAML or other type initialization exception";
+        public const string AmbiguousSelectedTab = "The requested selected tab matches more than one child page";
```

Comment:
It appears that the changes to the NavigationException don't seem to have been implemented anywhere... we'll need to locate the appropriate place in the NavigationService to add the new AmbiguousPathException as the inner exception for this use case

Reply:
I don't think that this is needed since Prism will just navigate to the first matching Tab and continue navigating within that tab.

In [your update](https://brianlagunas.com/whats-new-in-prism-for-xamarin-forms-7-0/) for Prism 7.0 you wrote, that having multiple NavigationPages is not an issue anymore since we are using the selectedTab parameter.